src/wam2layers/tracking/forwardtrack.py

```diff
@@ -92,16 +98,10 @@ def forwardtrack(
     # at this point any of the storages could still be overfull, thus stabilize and assigns losses:
```

Comment:
At this point the storages could still be overfull _internally_, thus stabilize and assign _internal_ losses